ekip/everykid/templates/about.html

```diff
@@ -149,6 +149,10 @@
 
       <h2>Code</h2>
       <p>We developed this website in the <a href="https://github.com/18F/ekip-api">open</a> and welcome your contributions and comments.</p>
+    
+      <h2>Contact us</h2>
+      <p>everykidinapark@ios.doi.gov</p>
```

Comment:
I think we should wrap this in a mailto: 
Also, should we add a sentence on what things people can contact that email address with?